_includes/manuals/1.18/3.5.5_facts_and_the_enc.md

```diff
@@ -28,6 +28,7 @@ Unless it already exists from setting up reporting, create a new configuration f
 </pre>
 
 Edit the URL field to point to your Foreman instance, and the SSL fields for the hostname of the Puppet master (which may be the same host). Paths to Puppet's SSL certificates will be under /etc/puppetlabs/ssl/ when using Puppet 4 with AIO. More information on SSL certificates is at [Securing communications with SSL](/manuals/{{page.version}}/index.html#5.4SecuringCommunicationswithSSL).
+When using Puppet 5 with AIO, puppetdir should be set to /opt/puppetlabs/server/data/puppetserver/
```

Comment:
Do you think we could just update the example to use Puppet 4 (and later) AIO paths? Or provide 2 examples, a p3 and p4 version?

Reply:
I did this because I don't know if Puppet 4 AIO use the same path. I didn't find a clear documentation about this on puppet doc. Just know that this works for me.

Reply:
They do indeed use the same paths, and since P3 is pretty old, it probably makes sense to update the example

Reply:
Please also update the 1.19 and nightly manual with the correct paths, thanks!